Property_map/doc/Property_map/Doxyfile.in

```diff
@@ -2,4 +2,5 @@
 
 PROJECT_NAME = "CGAL ${CGAL_DOC_VERSION} - CGAL and Boost Property Maps"
 INPUT += ${CGAL_PACKAGE_INCLUDE_DIR}/CGAL/property_map.h
+EXCLUDE += ${CGAL_PACKAGE_INCLUDE_DIR}/CGAL/Dynamic_property_map.h
```

Comment:
I think
```C++
#ifndef DOXYGEN_RUNNING
```
should be preferred: if the file is renamed or move, during the review for example, then it would remain outside the documentation.